src/Parser/Paladin/Retribution/Modules/Items/AshesToDust.js

```diff
@@ -8,7 +8,7 @@ import Module from 'Parser/Core/Module';
 import Combatants from 'Parser/Core/Modules/Combatants';
 import Enemies from 'Parser/Core/Modules/Enemies';
 
-import GetDamageBonus from '../PaladinCore/GetDamageBonus';
+import GetDamageBonus from '../../../Shared/GetDamageBonus';
```

Comment:
The import strings on all of the "Shared" modules should get updated to remove the ``../../../``